### PR TITLE
Bug 2039619: - [AWS] In tree provisioner storageclass aws disk type should contain 'gp3' and csi provisioner storageclass default aws disk type should be 'gp3'

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before:
Screen Shot 2022-01-14 at 9 34 37 AM

Screen Shot 2022-01-14 at 9 35 06 AM

### After:
Screen Shot 2022-01-14 at 9 07 36 AM

Screen Shot 2022-01-14 at 8 56 37 AM

### CODE REVIEW

File: frontend/public/components/storage-class-form.tsx [hunk 1]
```diff
@@ -171,7 +171,7 @@ class StorageClassFormWithTranslation extends React.Component<
       parameters: {
         type: {
           name: 'Type', // t('public~Type')
-          values: { io1: 'io1', gp2: 'gp2', sc1: 'sc1', st1: 'st1' },
+          values: { io1: 'io1', gp2: 'gp2', gp3: 'gp3', sc1: 'sc1', st1: 'st1' },
```
Comment (jhadvig) [inline frontend/public/components/storage-class-form.tsx:174]: I think the `gp3` should be first, in order to be the default

**Review by jhadvig — CHANGES_REQUESTED** [review]

File: frontend/public/components/storage-class-form.tsx (hunk 1 above)
Comment (cyril-ui-developer) [reply frontend/public/components/storage-class-form.tsx:174]: @jhadvig The expected result for the issue in BZ didn't state `Default is gp3` for `kubernetes.io/aws-ebs`. I have updated the default for `ebs.csi.aws.com`. However I noticed the user still needs to select the default value for it to be selected. Other fields like Filesystem Type behave the same way.

Comment (jhadvig) [reply frontend/public/components/storage-class-form.tsx:174]: Yes, sorry you are right, on the CSI shuold default to GP3.

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Pull request merged**